scratch.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+from matplotlib import pyplot as plt
+
+
+def draw_circle_segment(image, origin, radius, heading, theta):
```

Comment:
nothing called "scratch" - delete this? move these tools elsewhere? 

also needs a license

Reply:
@alexsax i think this is yours?